WootricSDK/WootricSDK/UIButton+Extension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIButton+Extension.swift
+//  WootricSDK
+//
+//  Created by Vijay Radake on 11/05/22.
+//  Copyright © 2022 Wootric. All rights reserved.
+//
+
+import Foundation
+
+@objc public extension UIButton
+{
+    func configureContentInsets(top: CGFloat, leading: CGFloat, bottom: CGFloat, trailing: CGFloat)
+    {
+//        if #available(iOS 15.0, *) {
```

Comment:
Can we remove the commented code? 🙏

Reply:
Yes we can. This code is for future releases so I kept it commented.